packages/loader/container-loader/src/container.ts

```diff
@@ -1615,8 +1615,6 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         const outboundMessageType: string = type;
         switch (outboundMessageType) {
             case MessageType.Operation:
-            case MessageType.RemoteHelp:
```

Comment:
Is this intentional? This will now fall-through to the summarize op, right?

Reply:
That was a mistake :)